src/main/scala/org/deepdive/settings/SettingsParser.scala

```diff
@@ -1,253 +1,536 @@
 package org.deepdive.settings
```

Comment:
Larger functions are no doubt bad, but I don't think that necessarily applies to files.  I found it much easier to stare at one file than getting lost while trying to guess where something might exist.  Since Scala doesn't force us to sync the filenames with its contents, I think it's better to keep the number of files minimal.  The only downside I see is slower compilation due to putting too many independent classes in the same file.  But I believe settings classes don't need frequent changes.  Does this sound convincing?